### PR TITLE
"has accepted handshake" indicator

### PR DESCRIPTION
**Pull request by mjoyce91**
Merge #584 first

Adds a new field, `has_accepted_other_offer`, which indicates whether the bidder has accepted a handshake for a cycle position that is in an active bid cycle

Note:
- you'll need to re-install dependencies
- you'll need to run migrations

### CODE REVIEW

File: talentmap_api/fsbid/services/reference.py [hunk 1]
```diff
@@ -4,11 +4,32 @@
...
+from talentmap_api.fsbid.views import reference as views
+
 API_ROOT = settings.FSBID_API_URL
 
 logger = logging.getLogger(__name__)
 
 
+def get_cycles(jwt_token):
+    '''
+    Gets an indivdual available position by id
```
Comment (elizabeth-jimenez) [inline talentmap_api/fsbid/services/reference.py:17]: 👀

File: talentmap_api/fsbid/services/bureau.py [hunk 2]
```diff
@@ -159,6 +162,14 @@ def fsbid_bureau_position_bids_to_talentmap(bid, jwt, cp_id, active_perdet):
     fullname = bid.get("full_name", None)
     if fullname:
         fullname = fullname.rstrip(' Nmn')
+    
+    hasAcceptedOffer = False
+    cycles = get_cycles(jwt)
+    cycles = pydash.map_(cycles, 'id')
```
Comment (elizabeth-jimenez) [inline talentmap_api/fsbid/services/bureau.py:168]: So i may learn: So the cycles in `cycles` table are always active?

**Comment by elizabeth-jimenez** [comment]
Just a couple things, but mostly I want to play with the caching before this is merged 😁

File: talentmap_api/fsbid/services/common.py [hunk 3]
```diff
@@ -481,7 +490,14 @@ def archive_favorites(ids, request, isPV=False, favoritesLimit=FAVORITES_LIMIT):
 def has_competing_rank(jwt, perdet, pk):
     rankOneBids = AvailablePositionRanking.objects.filter(bidder_perdet=perdet, rank=0).exclude(cp_id=pk).values_list(
         "cp_id", flat=True)
-    for y in rankOneBids:
+    rankOneBids = list(rankOneBids)
+    aps = []
+    if rankOneBids:
+        ids = ','.join(rankOneBids)
+        ap = apservices.get_available_positions({ 'id': ids, 'page': 1, 'limit': len(rankOneBids) or 1 }, jwt)
```
Comment (elizabeth-jimenez) [inline talentmap_api/fsbid/services/common.py:497]: Oh, is this to filter out the rankOneBids not in an active cycle?

**Review by elizabeth-jimenez — APPROVED** [review]
🍬 👍 Cool.Job

File: talentmap_api/fsbid/services/bureau.py (hunk 2 above)
Comment (mjoyce91) [reply talentmap_api/fsbid/services/bureau.py:168]: No, there could be inactive ones in there from the past

File: talentmap_api/fsbid/services/common.py (hunk 3 above)
Comment (mjoyce91) [reply talentmap_api/fsbid/services/common.py:497]: yes

**Pull request merged**